src/navigation/tabNavigation.tsx

```diff
@@ -18,35 +18,41 @@ export const TabNavigation = () => {
 
   return (
     <NavigationContainer>
-      <Tab.Navigator>
+      <Tab.Navigator
+        screenOptions={{
```

Comment:
move screen options of navigator and options of screens to a separate variables or separate files